runner.py

```diff
@@ -670,10 +688,15 @@ def get_main_parser():
                 job_script_prologue=job_script_prologue,
             )
         elif "condor" in args.executor:
+            portopts = {}
+            if "brux" in args.executor:
+                import socket
+                portopts = {"port": 8786, "host": socket.gethostname()} 
             cluster = HTCondorCluster(
                 cores=args.workers,
                 memory=f"{args.memory}GB",
                 disk=f"{args.disk}GB",
+                scheduler_options=portopts,
                 job_script_prologue=job_script_prologue,
```

Comment:
Please also add the ` job_extra_directives`  https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/58/files#diff-34cde5e49b54d9b67952c80d3580b6c421ebcf16097996e215aef54e44f78562R667-R674

Reply:
It ran on brux without `job_extra_directives`. Probably not needed